chapters/ch07.asciidoc

```diff
@@ -1419,9 +1419,9 @@ const things = {
   length: 2
 }
 Array.from(things)
-// <- [<div>, <div>, <div>, ...]
+// <- [Object, Object]
```

Comment:
Maybe we should extract the `apple` and `onion` to variables by those names, then reference the variables in the comment, for clarity?